allegro.h

```diff
@@ -613,7 +617,7 @@ typedef class Serial_write_buffer: public Serial_buffer {
         assert(ptr < fence);
         // two brackets surpress a g++ warning, because this is an
         // assignment operator inside a test.
-        while ((*ptr++ = *s++)) assert(ptr < fence);
+        while ((*ptr++ = *s++)) { assert(ptr < fence); }
```

Comment:
If you want to be strict about formatting, put a line break after { and another before }.

Reply:
My intention was not to be strict about aesthetic formatting, but explicit about what statements belong to the `while`, as one of the other warnings (`misleading-indentation`) was caused by similar formatting. I will go ahead and apply your suggestion though.